### PR TITLE
write readme file

### LINKED ISSUE #12: write readme file

### PR DESCRIPTION
**Pull request by A7medhq**
relates #12

Time taken: 3 hours

### CODE REVIEW

**Review by kakashii7 — APPROVED** [review]
Nice 
Gj

File: README.md [hunk 1]
```diff
@@ -1,2 +1,144 @@
...
+**User journey**
+
+1. Openening the application the user is going to see different categories and a collection of random images that he can scroll through.
+
+1. He should be able to choose any image from the random images that appear in the home-page.
+
+1. Then he shall be referred to another page containing the image he choosed to download it to his device.
+
+1. back to the home page he should be able to choose any cat. He is interested in from the home-page to navigate to the category section then to the download.
+
```
Comment (Ibrahim-Jarada) [inline README.md:100]: cat. its preferable to write the full-word  in order not to misunderstood by other words

File: README.md [hunk 2]
```diff
@@ -1,2 +1,144 @@
...
+<br><br>
+
+
+
+# ♠ Features <span id="features"></span>
+
+* As a user, you'll be able to download any image as a wallpaper to your phone, tablet or desktop.
+* As a user, you'll be able to choose any category in which you choose the wallpaper.
+* As a user, you'll be able to generate a random wallpaper with a random image and a random on it.
+* As a user, you'll be able to download the image with the caption as one photo.
```
Comment (Ibrahim-Jarada) [inline README.md:114]: * As a user, you'll be able to generate a random wallpaper with a random image and a random on it.

you missed the word quote

=> and a random **quote** on it.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,144 @@
...
+* As a user, you'll be able to change the caption alone if you don't like it.
+* As a user, you'll be able to write your own caption.
+* As a user, you'll be able to change the image alone if you don't like it.
+* As a user, you'll be able to save any combination of images and captions in the favorites secrion.
```
Comment (Ibrahim-Jarada) [inline README.md]: typo=>favorites **secrion**

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]
Can you add these fields?

The problem
Let people know what the problem that your project will solve and describe it.

The solution ( description of your website)
Let people know what your project can do specifically. Provide context and add a link to any reference visitors might be unfamiliar with. A list of Features or a Background subsection can also be added here. If there are alternatives to your project, this is a good place to list differentiating factors.

The wireframes ( The initial screens)
Let people know your planning process and the initial screens that you agreed on it then upload or add your sketch you can use balsamiq wireframe

[readme-tips](https://github.com/gazaskygeeks/Coding-Foundations-course/blob/master/coursebook/Week%2007/notes/readme-tips.md)

**Review by IsraaAmani — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]
Well-done

**Pull request merged**